MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.PasswordBox.xaml

```diff
@@ -17,7 +17,7 @@
         <Setter Property="BorderThickness" Value="0 0 0 1"/>
         <Setter Property="wpf:TextFieldAssist.TextBoxViewMargin" Value="1 0 0 0" />
         <Setter Property="Background" Value="Transparent"/>
-        <Setter Property="CaretBrush" Value="{Binding RelativeSource={RelativeSource Self}, Path=BorderBrush}"/>
+        <Setter Property="CaretBrush" Value="{Binding Path=(wpf:TextFieldAssist.UnderlineHighlightedBrush), RelativeSource={RelativeSource Self}}" />
```

Comment:
I don't think we should re-use the underline brush here. I think we should fix up the `PasswordBox` so that it matches the TextBox and uses `{DynamicResource PrimaryHueMidBrush}`